x-pack/metricbeat/module/sql/doc.go

```diff
@@ -0,0 +1,6 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// Package sql is a Metricbeat module that contains MetricSets.
```

Comment:
Change the description of the package.
```suggestion
// Package sql is a Metricbeat module that collects information from databases using SQL queries.
```

Reply:
Done.